platform-includes/getting-started-primer/javascript.nuxt.mdx

```diff
@@ -1,4 +1,4 @@
 <Alert level="warning">
-  This SDK is considered **experimental and in an alpha state**. It may experience breaking changes. Please reach out on
+  This SDK is **in an beta state**. It may experience breaking changes. Please reach out on
```

Comment:
beta technically means, we don't break APIs any more. 

```suggestion
  This SDK is **in an beta state**. It may still experience changes in behavior. Please reach out on
```